nodes/parachain/src/service.rs

```diff
@@ -270,11 +271,13 @@ where
 	let rpc_builder = {
 		let client = Arc::clone(&client);
 		let transaction_pool = Arc::clone(&transaction_pool);
+		let backend = Arc::clone(&backend);
 
 		Box::new(move |_| {
 			let deps = crate::rpc::FullDeps {
 				client: Arc::clone(&client),
 				pool: Arc::clone(&transaction_pool),
+				backend: Arc::clone(&backend),
```

Comment:
You need to add `ismp-parachain-inherent` to the`create_inherent_providers` in `AuraParams`
guide:  https://docs.hyperbridge.network/developers/polkadot/ismp-parachain-inherent